src/browser/preload_scripts.ts

```diff
@@ -191,7 +191,6 @@ function loadFromCache(opts: FetchResponse): Promise<boolean> {
             const id = getIdentifier(preload);
 
             logPreload('info', identity, 'load started', id);
```

Comment:
should the logPreload be removed too

Reply:
It's complicated... right now we'll see this "fetch to in memory cache" logging, if I remove we won't see it at all. but it's a duplicate at the same time... so not sure.
Alternatively, this will be removed and cleaned up when I will be cleaning up preload script file later...